test/sagemaker_tests/pytorch/training/integration/sagemaker/test_gdrcopy.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+import pytest
+from sagemaker import utils
+from test.test_utils import get_framework_and_version_from_tag, get_cuda_version_from_tag
+from packaging.version import Version
+from packaging.specifiers import SpecifierSet
+from ...integration import (
+    DEFAULT_TIMEOUT,
+)
+from ...integration.sagemaker.timeout import timeout
+from . import invoke_pytorch_estimator
+
```

Comment:
The get_efa_test_instance_type function needs to be imported

Reply:
```suggestion
from ...integration import (
    DEFAULT_TIMEOUT,
)
from ...integration.sagemaker.timeout import timeout
from . import invoke_pytorch_estimator
from ....training import get_efa_test_instance_type

```

Reply:
Yeah I also noticed that when porting the fixes to 1.13.1. Thanks!

Reply:
done